Common/Securities/Option/OptionStrategies.cs

```diff
@@ -912,6 +912,186 @@ public static OptionStrategy ShortBoxSpread(Symbol canonicalOption, decimal high
+        /// <param name="nearExpiration">Near expiration date for the short call and the long put</param>
+        /// <param name="farExpiration">Far expiration date for the long call and the short put</param>
+        /// <returns>Option strategy specification</returns>
+        public static OptionStrategy JellyRoll(Symbol canonicalOption, decimal strike, DateTime nearExpiration, DateTime farExpiration)
+        {
+            var callCalendarSpread = CallCalendarSpread(canonicalOption, strike, nearExpiration, farExpiration);
+            var shortPutCalendarSpread = ShortPutCalendarSpread(canonicalOption, strike, nearExpiration, farExpiration);
+
+            return new OptionStrategy
+            {
+                Name = OptionStrategyDefinitions.JellyRoll.Name,
+                Underlying = canonicalOption.Underlying,
+                CanonicalOption = canonicalOption,
+                OptionLegs = callCalendarSpread.OptionLegs.Concat(shortPutCalendarSpread.OptionLegs).ToList()
+            };
+        }
+
+        /// <summary>
+        /// Creates new Short Jelly Roll strategy which combines a long call calendar spread and a short put calendar spread
+        /// with the same strikes and the same pair of expiration dates.
+        /// </summary>
+        /// <param name="canonicalOption">Option symbol</param>
+        /// <param name="strike">The strike price of the all legs</param>
+        /// <param name="nearExpiration">Near expiration date for the short call and the long put</param>
+        /// <param name="farExpiration">Far expiration date for the long call and the short put</param>
+        /// <returns>Option strategy specification</returns>
+        public static OptionStrategy ShortJellyRoll(Symbol canonicalOption, decimal strike, DateTime nearExpiration, DateTime farExpiration)
+        {
+            return InvertStrategy(JellyRoll(canonicalOption, strike, nearExpiration, farExpiration), OptionStrategyDefinitions.ShortJellyRoll.Name);
+        }
+
+        /// <summary>
+        /// Method creates new Bear Call Ladder strategy, that consists of three calls with the same expiration but different strikes.
+        /// The strike price of the short call is below the strikes of the two long calls.
+        /// </summary>
+        /// <param name="canonicalOption">Option symbol</param>
+        /// <param name="leg1Strike">The strike price of the short call</param>
+        /// <param name="leg2Strike">The strike price of the long call with lower strike price</param>
+        /// <param name="leg3Strike">The strike price of the long call with higher strike price</param>
+        /// <param name="expiration">Option expiration date</param>
+        /// <returns>Option strategy specification</returns>
+        public static OptionStrategy BearCallLadder(
+            Symbol canonicalOption,
+            decimal leg1Strike,
+            decimal leg2Strike,
+            decimal leg3Strike,
```

Comment:
Believe `higherStrike/middleStrike/lowerStrike` would be better names, like we use in the other strategies, same comment for the other ones